packages/remix-validated-form/src/server.ts

```diff
@@ -33,7 +34,7 @@ export function validationError(
       repopulateFields,
       formId: error.formId,
     },
-    { status: 422 }
+    { ...init, status: init.status || 422 }
```

Comment:
` init.status` is causing a type error since init can be undefined. Could we change this to this?

```ts
{ status: 422, ...init }
```

Reply:
[Done](https://github.com/airjp73/remix-validated-form/pull/90/commits/71847ea80bf2b4cc29601669701023adf4f231d7)